packages/react-components/src/organisms/__tests__/TeamGroupCard.test.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { createListGroupResponse } from '@asap-hub/fixtures';
+import userEvent from '@testing-library/user-event';
+
+import TeamGroupTools from '../TeamGroupCard';
+
+it('renders team group tools with 1 group', () => {
+  const { getByRole, queryByText } = render(
+    <TeamGroupTools {...createListGroupResponse(1)} />,
+  );
+  expect(getByRole('heading', { level: 3 }).textContent).toMatchInlineSnapshot(
+    `"Team Groups (1)"`,
+  );
+  expect(queryByText(/view (more|less)/i)).not.toBeInTheDocument();
+});
+
+it('renders team group tools with 2 group', () => {
+  const { getByRole, queryByRole } = render(
+    <TeamGroupTools {...createListGroupResponse(2)} />,
+  );
+  expect(getByRole('heading', { level: 3 }).textContent).toMatchInlineSnapshot(
+    `"Team Groups (2)"`,
+  );
+  expect(queryByRole('button')).not.toBeInTheDocument();
+});
+
+it('renders team group tools with 1 group and 1 team so team is singular', () => {
+  const { getByText } = render(
+    <TeamGroupTools {...createListGroupResponse(1, { teamsCount: 1 })} />,
+  );
+
+  expect(getByText('1 Team')).toBeInTheDocument();
+});
+it('renders team group tools with 1 group and 2 teams so teams is pluralized', () => {
+  const { getByText } = render(
+    <TeamGroupTools {...createListGroupResponse(1, { teamsCount: 2 })} />,
+  );
+  expect(getByText('2 Teams')).toBeInTheDocument();
+});
+it('renders team group tools with 3 groups so view more button is visable', () => {
+  const { queryByText } = render(
```

Comment:
```suggestion
  const { getByText } = render(
```

Reply:
The error message when using query is really shit (because it's just asserting on null)